src/requirements.md

```diff
@@ -23,6 +23,14 @@ Note: The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “
   - The client MUST contain standalone documentation for users
   - The client library code SHOULD contain inline documentation for current and future developers
 
+## The runtime environment
```

Comment:
Sorry, but I didn't get the wording. Runtime itself is a piece of WebAssembly Code. In #18 you state that we are going to use bynaryen to perform runtime calls (with the correct callbacks from wasm to c++ implemented). I would move these points there as justification why we choose bynaryen to execute wasm

Reply:
Ok, I thought all the requirements should go here.

Reply:
In such case should there be anything added in this file?

Reply:
Not sure, it rather more about design choice than about requirement

Reply:
Ok, I'll move the points to the implementation file under the runtime details PR and close this PR then. Is that ok?

Reply:
Sure, PRs shouldn't be a bureaucracy, any suitable to you option should work